### PR TITLE
Regroup imports

### PR DESCRIPTION
**Pull request by yaahc**
I was able to hack a version of rustfmt locally and just tweaked the one line that prevented it from grouping use statements with newlines between them back together and ran that once to get this diff. No rush on this one, but I figured it would be good to toss it together quickly for when we talk about import style in our next gardening session.

### CODE REVIEW

File: zebra-chain/src/block/tests/vectors.rs [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-};
-
-use chrono::{DateTime, Duration, LocalResult, TimeZone, Utc};
-
-use crate::serialization::{sha256d, ZcashDeserialize, ZcashDeserializeInto, ZcashSerialize};
-use crate::transaction::LockTime;
-
-use super::super::{serialize::MAX_BLOCK_BYTES, *};
-use super::generate; // XXX this should be rewritten as strategies
+}; // XXX this should be rewritten as strategies
```
Comment (yaahc) [inline zebra-chain/src/block/tests/vectors.rs:9]: cute comment behaviour

**Review by yaahc — COMMENTED** [review]
laughing at how many merge conflicts this would create if merged.

**Comment by hdevalence** [comment]
I really don't think that we should implement custom rustfmt rules for use in our project.

**Comment by yaahc** [comment]
> I really don't think that we should implement custom rustfmt rules for use in our project.

I haven't implemented any rustfmt rules, it was a onetime hack to see how hard it would be to get our repo into a consistent state once where we could go back to maintaining our preferred style via code review after. Also, I would it help if I said I did this on my own time?

**Pull request closed without merging**